lib/plug/ssl.ex

```diff
@@ -52,20 +54,22 @@ defmodule Plug.SSL do
   """
   @behaviour Plug
 
+  require Logger
   import Plug.Conn
   alias Plug.Conn
 
   def init(opts) do
-    {hsts_header(opts), Keyword.get(opts, :host), Keyword.get(opts, :rewrite_on, [])}
+    {hsts_header(opts), Keyword.get(opts, :host), Keyword.get(opts, :rewrite_on, []),
```

Comment:
Maybe refactor this to:

```elixir
host = Keyword.get(opts, :host)
rewrite_on = Keyword.get(opts, :rewrite_on, [])
log_level = Keyword.get(opts, :log, :info)
{hsts_header(opts), host, rewrite_on, log_level}
```

It's a personal preference but I think this way it's easier to read. :)

Reply:
Agreed, just followed the format in `Plug.Logger`.